CMakeLists.txt

```diff
@@ -43,6 +43,13 @@ if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
   set(CMAKE_INSTALL_PREFIX "/opt/intel/ipcl")
 endif()
 
+if(CMAKE_PREFIX_PATH)
+  message(STATUS "---- CMAKE_PREFIX_PATH is defined as ${CMAKE_PREFIX_PATH}")
+else()
+  set(CMAKE_PREFIX_PATH $ENV{HOME}/intel /opt/intel)
```

Comment:
Is this to read the dependencies that are installed during IPCL build/installation (CMAKE_INSTALL_PREFIX)?